src/cephadm/cephadm

```diff
@@ -3921,16 +3921,23 @@ def command_bootstrap(ctx):
     # create mgr
     create_mgr(ctx, uid, gid, fsid, mgr_id, mgr_key, config, cli)
 
+    def json_loads_retry(cli_func):
+        for sleep_secs in [1, 4, 4]:
+            try:
+                return json.loads(cli_func())
+            except:
+                logger.debug('Invalid JSON. Retrying in %s seconds...' %  sleep_secs)
+                time.sleep(sleep_secs)
+
```

Comment:
```
       return json.loads(cli_func())
```
Should I add the above on line 3931 so that we call one more time and if the exception is thrown at least the error is identifcal so as to not introduce any new behavior?